src/net/sf/memoranda/ui/WorkPanel.java

```diff
@@ -225,10 +230,19 @@ else if (pan.equals("TASKS"))
 				tasksB_actionPerformed(null);
 			else if (pan.equals("EVENTS"))
 				eventsB_actionPerformed(null);
-			else if (pan.equals("FILES"))
-				filesB_actionPerformed(null);
+			else if (pan.equals("FILES")) {
+					filesB_actionPerformed(null);
```

Comment:
going forward, you're absolutely correct. however, if the user was last on resources pane before they get this update, they'll run into this situation.